### ISSUE TITLE
[Chore] Ruby 2.7.0 changes

### ISSUE THREAD

**Issue by noraj** [socketry/async-http#42]
```
/home/noraj/.rbenv/versions/2.7.0/lib/ruby/gems/2.7.0/gems/async-http-0.38.3/lib/async/http/url_endpoint.rb:163: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

Ruby 2.7

- Quick changelog: https://www.ruby-lang.org/en/news/2019/12/25/ruby-2-7-0-released/
- Comprehensive changelog: https://rubyreferences.github.io/rubychanges/2.7.html
- Ruby 2.7 NEWS: Commentary by Cookpad’s Full Time Ruby Committers: https://sourcediving.com/ruby-2-7-news-commentary-by-cookpads-full-time-ruby-comitters-bdbaacb36d0c

The warning must be due to _Separation of positional and keyword arguments_

Related issue https://github.com/picatz/shodanz/issues/32

**Comment by ioquatix** [comment]
I believe this issue is already fixed in `async-http` 0.50.0

**Comment by noraj** [comment]
You're right I'm using outdated versions.

Creating a simple Gemfile

```ruby
source 'https://rubygems.org'

gem 'shodanz', '>=2.0.1'
gem 'async-http', '>=0.50'
gem 'async-io',   '~>1.27'
```

And executing with `bundle exec ruby` make it right.
